testsuite/integration-arquillian/tests/base/src/test/resources/export/partialexport-testrealm.json

```diff
@@ -1155,6 +1155,9 @@
           ],
           "authType": [
             "simple"
+          ],
+          "enabled": [
+            "false"
```

Comment:
Good question, I am not sure it is necessary any more. I added this because when I added email/username uniqueness check into map adapter import test started to fail, because there is LDAP configured, which means when a user was imported, the check was called, and it tried to check for ldap users with the given email, that failed because ldap is not running it just checks whether ldap configuration was imported. 

However, since we changed the check to just check current storage, I think we can remove this.